Packages/Tracking/Core/Runtime/Scripts/MRTK Support/LeapMRTKSubsystem.cs

```diff
@@ -231,12 +231,12 @@ private static void RunAfterSceneLoad()
                     return;
                 }
 
-                LeapProvider leapProvider = GameObject.FindAnyObjectByType<LeapXRServiceProvider>();
```

Comment:
This would mean users with PostProcessProviders would find it doesn't work for them, but as it currently doesn't at all I think your suggestion is a good one to at least improve things for now. So yes agreed.